packages/flutter_tools/test/project_test.dart

```diff
@@ -17,8 +17,7 @@ void main() {
       final Directory directory = fs.directory('myproject');
       expect(
         new FlutterProject(directory).directory.absolute.path,
-        directory.absolute.path,
-      );
+        directory.absolute.path,);
```

Comment:
Please put the `);` back to whence it came.

Reply:
ok, ok... will do );